src/main/java/core/basesyntax/imp/CustomFileReadImp.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.imp;
+
+import core.basesyntax.service.CustomFileReader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.LinkedList;
+import java.util.List;
+
+public class CustomFileReadImp implements CustomFileReader {
+    @Override
+    public List<String> readFromFile(String fileName) {
```

Comment:
to follow SRP make this class responsible only for reading from file, parse it records in different class